Voycar.Api.Web.Tests.Unit/Members/Endpoints/Get/Personal/Endpoint.cs

```diff
@@ -0,0 +1,95 @@
+namespace Voycar.Api.Web.Tests.Unit.Members.Endpoints.Get.Personal;
+
+using Entities;
+using FakeItEasy;
+using Features.Members.Endpoints.Get.Personal;
+using Features.Members.Repository;
+using Features.Users.Repository;
+using Microsoft.Extensions.DependencyInjection;
+
+
+public class Endpoint : TestBase<App>
+{
+    private readonly IUsers FakeUserRepository = A.Fake<IUsers>();
+    private readonly IMembers FakeMemberRepository = A.Fake<IMembers>();
+
+    private readonly Request Request = new();
+
+
+    private Features.Members.Endpoints.Get.Personal.Endpoint SetupEndpoint()
+    {
+        return Factory.Create<Features.Members.Endpoints.Get.Personal.Endpoint>(ctx =>
+        {
+            ctx.AddTestServices(s =>
+            {
+                s.AddSingleton(this.FakeUserRepository);
+                s.AddSingleton(this.FakeMemberRepository);
+            });
+        });
+    }
+
+
+    [Fact]
+    public async Task Get_Request_Throws_ValidationsFailure_DueToNullUser()
+    {
+        // Arrange
+        var ep = this.SetupEndpoint();
+
+        A.CallTo(() => this.FakeUserRepository.Retrieve(this.Request.UserId)).Returns(null);
+
+        // Act
+        async Task Act() => await ep.HandleAsync(this.Request, default);
+
+
+        // Assert
+        var exception = await Assert.ThrowsAnyAsync<ValidationFailureException>(Act);
+        Assert.NotNull(exception);
+        Assert.Equal("ThrowError() called! - User does not exist", exception.Message);
```

Comment:
Wie funktioniert das? Woher weißt du, dass das die exakte Exception Message ist, die zurückkommt? Das "ThrowError() called! - " haben wir doch nie irgendwo geschrieben

Reply:
Das ist die Error Message, die man bekommt, wenn man den Endpoint nicht mit einem Http Client aufruft. Warum genau das "ThrowError()" mit drinnen steht, weiß ich auch nicht, aber wenn man nur das "User does not exist" rein packt, dann erkennt er die Exception nicht.